.github/workflows/build_demos.yml

```diff
@@ -0,0 +1,50 @@
+name: Demos
+
+on:
+  pull_request:
+    branches: '**'
```

Comment:
is there no need to run it on push like post-commit for master or by schedule like nightly?

Reply:
Why would we need it?

Reply:
just for reliability that merge of pull request does not break anything (e.g. due to wrong rebase)

Reply:
but it is up to you, I do not insist